packages/perseus/src/widgets/dropdown/validate-dropdown.test.ts

```diff
@@ -0,0 +1,31 @@
+import validateDropdown from "./validate-dropdown";
+
+import type {PerseusDropdownUserInput} from "../../validation.types";
+
+describe("validateDropdown", () => {
+    it("returns invalid for invalid input (user input of 0)", () => {
```

Comment:
Tiny improvement suggestion as `0` is the selected index. 🤷‍♂️  You decide. :) 

```suggestion
    it("returns invalid when nothing selected (user input of 0)", () => {
```